packages/flutter/lib/src/widgets/form.dart

```diff
@@ -635,4 +697,13 @@ enum AutovalidateMode {
   /// Used to auto-validate [Form] and [FormField] only after each user
   /// interaction.
   onUserInteraction,
+
+  /// Used to auto-validate current [FormField] when unfocused.
+  ///
+  /// This mode is useful when you want to validate a [FormField] after the user
+  /// has interacted with another one.
+  ///
```

Comment:
I think I'd remove this whole paragraph, it's kind of self-evident from the name onUnfocus.